docs/excel/custom-functions-overview.md

```diff
@@ -360,6 +360,50 @@ function secondHighest(values){
 }
 ```
 
+## Discovering cells that invoke custom functions
+
+You may wish to discover the cell that invoked a custom function. Common scenarios for using `caller.address` include:
```

Comment:
Intro is a little sparse, consider adding a sentence or two to describe why the reader is here or describe the problem and solution a bit more. 1-3 sentences is fine, but it needs more context.